config/auth.ts

```diff
@@ -232,8 +232,11 @@ const authConfig: AuthConfig = {
       driver: 'jwt',
       publicKey: Env.get('JWT_PUBLIC_KEY', '').replaceAll('\\n', '\n'),
       privateKey: Env.get('JWT_PRIVATE_KEY', '').replaceAll('\\n', '\n'),
-      persistJwt: false,
-      jwtDefaultExpire: '10m',
+      persistJwt: true,
+      // TODO: We should improve the following implementation as this is a security concern.
+      // The following ts-expect-error is to set exp to undefined (JWT with no expiration)
+      // @ts-expect-error
+      jwtDefaultExpire: undefined,
```

Comment:
Let's not forget about this TODO before merging this PR

Reply:
@mcmxcdev actually, our current implementation (with the previous version of the server) doesn't have an expiration to the JWT - look for the localStore authToken JWT and decode it.

I think that, for now, we should merge as it is, and then we make improvements as we go (I just left that TODO so we dont forget to address this issue - this needs to be changed both on the server and on the client)